lingvo/tasks/asr/encoder.py

```diff
@@ -64,9 +64,11 @@ def Params(cls):
     p.Define('input_shape', [None, None, None, None],
```

Comment:
Are you still using your changes to encoder.py or are all of your current models using encoder_v2.py?

Reply:
I have a few classes with encoder (including the 17.9 WER baseline) and most of the newer ones are with encoder v2.